### PR TITLE
updated mailto links to display digitalgov@gsa.gov in link copy

### PR DESCRIPTION
**Pull request by nick-mon1**
This PR implements the following **changes:**

Updated all events pages to display "digitalgov@gsa.gov" in the link text for users to copy. 
This changes "send us an email >>" to include the email address.

Source: [Investigate error on link for "Send us an email »"](https://trello.com/c/ZYmctzCZ)

Screen Shot 2022-05-17 at 9 35 06 AM

### CODE REVIEW

**Comment by afeijoo** [comment]
LGTM

File: themes/digital.gov/layouts/partials/core/page-data.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
...
+  <p class="feedback-copy"><i class="fas fa-question-circle"></i> {{ if .prompt -}}{{- .prompt -}}{{- else -}}Have feedback or questions?{{- end }} <a target="new" href="mailto:{{- if .email -}}{{- .email -}}{{- else -}}digitalgov@gsa.gov{{- end -}}?subject=Feedback%3A%20{{- htmlUnescape .page.Title | safeHTML -}}&amp;body=%0D%0A---enter%20your%20feedback%2Fquestions%20here---%0D%0A%0D%0A%0D%0A-%20-%20-%20-%20-%20%0D%0APage%3A%20{{- htmlUnescape .page.Title | safeHTML -}}%0D%0AURL%3A%20https%3A%2F%2Fwww.digital.gov{{- .page.Permalink | markdownify -}}%0D%0A-%20-%20-%20-%20-%0D%0A%0D%0A%0D%0A{{- $email_copy -}}%0D%0A%0D%0Ahttps%3A%2F%2Fwww.digital.gov%0D%0A%40digital_gov%20on%20Twitter{{- if .cc -}}&amp;cc={{- .cc -}}{{- end -}}">Send&nbsp;us&nbsp;an&nbsp;email&nbsp;at&nbsp;digitalgov@gsa.gov</a></p>
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/page-data.html:16]: Can we add some formatting to make this more readable?

Also, I don't think we need all of the `&nbsp;` in the link text.

**Review by mejiaj — CHANGES_REQUESTED** [review]
Looks good, the email is much clearer.

Just a suggestion on formatting, adding indents and removing the non-breaking spaces from the link text.

**Comment by nick-mon1** [comment]
Extracted the lengthy mailto url into variables for easier readability.

**Review by mejiaj — CHANGES_REQUESTED** [review]
The new formatting looks good, but needs another pass. The email is now showing up blank in outlook.

**Before**
image

**After**
image

```diff
{{/*  Old value  */}}
-<a target="new" href="mailto:digitalgov@gsa.gov?subject=Feedback%3A%20How%20to%20Build%20Your%20Digital%20Dream%20Team&amp;body=%0D%0A---enter%20your%20feedback%2Fquestions%20here---%0D%0A%0D%0A%0D%0A-%20-%20-%20-%20-%20%0D%0APage%3A%20How%20to%20Build%20Your%20Digital%20Dream%20Team%0D%0AURL%3A%20https%3A%2F%2Fwww.digital.gov%2f%2flocalhost%3a1313%2fevent%2f2022%2f05%2f17%2fhow-to-build-your-digital-dream-team%2f%0D%0A-%20-%20-%20-%20-%0D%0A%0D%0A%0D%0AWe%20value%20your%20feedback%20and%20open%20questions.%20Thank%20you%20for%20helping%20to%20make%20our%20pages%20better.%0D%0A%0D%0Ahttps%3A%2F%2Fwww.digital.gov%0D%0A%40digital_gov%20on%20Twitter&amp;cc=digitalgovu%40gsa.gov">Send&nbsp;us&nbsp;an&nbsp;email&nbsp;at&nbsp;digitalgov@gsa.gov</a>

{{/*  New value  */}}
+<a target="new" href="mailto:digitalgov@gsa.gov?subject=Feedback%3A%20How%20to%20Build%20Your%20Digital%20Dream%20Team&amp;amp;body=%0D%0A---enter%20your%20feedback%2Fquestions%20here---%0D%0A%0D%0A%0D%0A-%20-%20-%20-%20-%20%0D%0APage%3A%20How%20to%20Build%20Your%20Digital%20Dream%20Team%0D%0AURL%3A%20https%3A%2F%2Fwww.digital.gov//localhost:1313/event/2022/05/17/how-to-build-your-digital-dream-team/%0D%0A-%20-%20-%20-%20-%0D%0A%0D%0A%0D%0AWe%20value%20your%20feedback%20and%20open%20questions.%20Thank%20you%20for%20helping%20to%20make%20our%20pages%20better.%0D%0A%0D%0Ahttps%3A%2F%2Fwww.digital.gov%0D%0A%40digital_gov%20on%20Twitter&amp;cc=digitalgovu@gsa.gov">
  Send us an email at digitalgov@gsa.gov
</a>
```

**Comment by nick-mon1** [comment]
Previous commit was not properly escaping URL encoded characters inside string variables and causing the email details to not be displayed, tested on mac mail and outlook with the new formatting.

Screen Shot 2022-05-18 at 3 50 43 PM
Screen Shot 2022-05-18 at 3 51 15 PM

**Review by mejiaj — APPROVED** [review]
Looks good, thank you!

**Pull request merged**